BASE/Test/ServerTelemetryChannel.Test/TelemetryChannel.Nuget.Tests/TelemetryChannel.Nuget.Tests.csproj

```diff
@@ -20,7 +20,7 @@
 
   <PropertyGroup Condition="$(OS) != 'Windows_NT'">
```

Comment:
this whole section need to be removed. This test is only required for net45. To avoid running them in linux, we can rename project to .Win.Tests.csproj, and use "*Win.Tests.proj" as a filter in Linux build yml file.
This would apply to any other test project which are only supposed to targe full framework like .NET45.
(This can be done separately from this PR)

Reply:
I removed that section. and later, as you suggested, we can check a new pipeline in windows.
Thanks for reviewing